private/bufpkg/bufprotoplugin/cmd/protoc-gen-go-apiclientconnect/main.go

```diff
@@ -282,111 +305,3 @@ func generateServiceFile(helper protogenutil.NamedHelper, plugin *protogen.Plugi
 	}
 	return nil
 }
-
-func handleGlobal(helper protogenutil.NamedHelper, plugin *protogen.Plugin, goPackageFileSets []*protogenutil.GoPackageFileSet) error {
```

Comment:
This stuff was completely unused AFAICT. So I decided to go ahead and remove it.